schemas/2016-03-01/Microsoft.NotificationHubs.json

```diff
@@ -0,0 +1,686 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.NotificationHubs.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.NotificationHubs",
+  "description": "Microsoft NotificationHubs Resource Types",
+  "resourceDefinitions": {
+    "namespaces": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.NotificationHubs/namespaces"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2016-03-01"
+          ]
+        },
+        "properties": {
+          "oneOf": [
```

Comment:
Yes, I do. I'll have to look and see why my changes didn't get your SKU changes.